Automattic/sensei
# Add WCPC installation notice

Fixes #2789

### Changes proposed in this Pull Request

* This PR introduces a WCPC prompt suggesting the user to install the plugin to sell their courses.

### Testing instructions

1. Create a scenario with the following conditions:
    * At least one course has been published.
    * WCPC is not installed.
    * WooCommerce is installed.
2. Go to the _WP-admin > Courses_ page (`/wp-admin/edit.php?post_type=course`), and make sure you see the WCPC installation notice.
3. Click on "Learn more" and make sure you go to the extension page.
4. Click on the "X" button to dismiss the notice. Make sure the notice doesn't appear after refreshing the page.
5. Remove the `sensei_dismiss_wcpc_prompt` option to see the notice again.
6. Click on "Install now", and make sure you can install the extension.

Also, make sure the events were logged as described in the [issue](https://github.com/Automattic/sensei/issues/2789#issuecomment-743250846).

Also, install a paid extension through the setup wizard, in a site with Jetpack (connected), and make sure the `sensei_setup_wizard_features_install_paid` event continue being logged properly.


### Deprecated Code

* `Sensei_Setup_Wizard::get_woocommerce_connect_data`. Use `Sensei_Utils::get_woocommerce_connect_data` instead.


### Screenshot / Video

The text and links/buttons had some tweaks after a conversation with @donnapep. This screenshot is the final result:

<img width="1207" alt="Screen Shot 2021-01-07 at 11 38 47" src="https://user-images.githubusercontent.com/876340/103905107-f34eff80-50dc-11eb-9cd0-a72d63929534.png">
